### DISCUSSION TITLE
Most Languages able to include private repos too

### DISCUSSION THREAD

**Discussion by sc0rp10n-py** [anuraghazra/github-readme-stats#1618] [Q&A] [answered]
Can this feature be implemented somehow?

**Comment by rickstaa** [comment, marked as answer]
@sc0rp10n-py Thanks for your question. This feature has indeed been implemented. See more information see https://github.com/anuraghazra/github-readme-stats#adding-private-contributions-count-to-total-commits-count. Let me know if you have any problems getting your private contributions to show up. Please be aware that it might take some hours for changes to be reflected on the cards (see [cache](https://github.com/anuraghazra/github-readme-stats#common-options)).

**Reply by sc0rp10n-py** [reply]
Hi, I know it's implemented for total commits count, but I was talking about for most used languages card. I tried doing the same url change for it, but didn't seem to work for me.

**Reply by rickstaa** [reply]
@sc0rp10n-py Ah for the other cards the private contributions should work out of the box. It works for me, they work both in my dev environment and my own vercel instance. You can try it out by using the following query in the [graphql explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    # fetch only owner repos & not forks
    repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
      nodes {
        name
        languages(first: 10, orderBy: {field: SIZE, direction: DESC}) {
          edges {
            size
            node {
              color
              name
            }
          }
        }
      }
    }
  }
}
```

You can then use the following in the query variables field.

```graphql
{"login": "sc0rp10n-py"}
```
However, if you're using the public vercel instance (i.e. https://github-readme-stats.vercel.app/), it can not work since we do not have access to your private repositories. As a result, we can not count the language bytes. I would therefore advise you to set up your vercel instance. A guide on how this is done can be found [here](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).

**Reply by sc0rp10n-py** [reply]
@rickstaa ah cool makes sense. Thanks for sharing the guide.
